purchase_request/models/stock_rule.py

```diff
@@ -52,9 +52,10 @@ def _make_pr_get_domain(self, values):
         :return: False
         """
         domain = (
-            ("state", "=", "draft"),
+            ("state", "=", "approved"),
```

Comment:
hmm are we sure about this change? Shouldn't you select both states? Or exclude approved?